llama_stack/distribution/store/registry.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import json
+
+from typing import Protocol
+
+import pydantic
+
+from llama_stack.distribution.datatypes import RoutableObjectWithProvider
+
+from llama_stack.providers.utils.kvstore import KVStore
+
+
+class Registry(Protocol):
```

Comment:
maybe we can just call it `DistributionRegistry` instead of having to alias it later. Causes unnecessary confusion for people reading the code?

Reply:
done.